Makefile

```diff
@@ -175,6 +175,16 @@ else
 endif
 endif
 
+package:
+	mkdir -p ../package
+	$(MAKE) -C ../maccore package
+	# copy .pkg, .zip and *updateinfo to the packages directory to be uploaded to storage
+	cp ../maccore/release/*.pkg ../package
+	-cp ../maccore/release/*.zip ../package
+	-cp ../maccore/release/*updateinfo ../package
+	-cp ../maccore/tests/*.zip ../package
+	-cp ../xamarin-macios/tests/*.zip ../package
+
```

Comment:
this seems unrelated to the `--no-sudo` option

Reply:
Right the pull request title wasn't all encompassing. This was a part of the first pull request I submitted but I opened a new one with fewer changes. I'm building on VSTS and it makes it simpler to have the macios packaging step inside the macios makefile. We could be deprecating release-scripts and I believe this is the only repo that uses this task.

Reply:
@joewstroman ok, please add this information to the commit message when merging. thanks!